rpc/connection.go

```diff
@@ -219,10 +223,12 @@ func (ct *ConnectionTransportTLS) Dial(ctx context.Context) (
 			resinit.ResInitIfDNSError(err)
 			return err
 		}
+		ct.log.Info("baseConn: %s; Calling Handshake", baseConn.LocalAddr())
```

Comment:
Do we still need these two logs, or were they just for debugging?

Reply:
I think it's still good to have them, for figuring out when/if we actually have a TCP dial delay.